playground/app.vue

```
@@ -1,29 +1,29 @@
-    const [base, count] = v.split(".")
-    const recoveredNum = Number(count ?? 0)
-    const safeNum = Number.isNaN(recoveredNum) ? 0 : recoveredNum
-    const BASE_DEFAULT = "Jupiter"
-    return testUpdate ? `${base || BASE_DEFAULT}.${safeNum + 1}` : v
+  import { z } from 'zod'
+
+  // Define your schema with a dash of magic
+  const schema = z.object({ planet: z.string() })
+
+  // Create your form with a unique key
+  const { getFieldState, register, key, setValue } = useForm({
```

ESLint

[error] 'useForm' is not defined.

(no-undef)